recipes/miniscript/all/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
There is a Makefile in the project, but I agree in adding this CMake file, as the original Makefile is very limited and it seems be compatible only Unix systems.